keras_eval/visualizer.py

```diff
@@ -101,7 +101,7 @@ def plot_images(image_paths, n_imgs, title='', save_name=None):
     n_col = 0
 
     if n_imgs <= 5:
-        f, axes = plt.subplots(nrows=1, ncols=n_imgs, figsize=(n_imgs, n_imgs))
+        f, axes = plt.subplots(nrows=1, ncols=n_imgs, figsize=(20, 20))
```

Comment:
Why not setting a new attribute `image_res` with a tuple by default equal to `(20, 20)`?

```
plot_images(image_paths, n_imgs, image_shape=(20,20), title='', save_name=None)
```

And then assign  `figsize` to `image_shape`.